config/locales/en.yml

```diff
@@ -882,8 +882,8 @@ en:
       middleware_deployment:       Middleware Deployment
       middleware_deployments:      Middleware Deployments
       evm_owner:                   EVM Owner
-      ext_management_system:       Cloud/Infrastructure Provider
-      ext_management_systems:      Cloud/Infrastructure Providers
+      ext_management_system:       Cloud/Infrastructure/Containers Provider
```

Comment:
> @zeari wouldn't this change effect other areas of UI?

Ah, your'e right.
But even so, containers providers have an 'ExtManagementSystem' for a while now....